README.md

```diff
@@ -1,6 +1,6 @@
 # frc-template
 
-[![CircleCI](https://circleci.com/gh/team5499/frc-template.svg?style=svg)](https://circleci.com/gh/team5499/frc-template)
+![TravisCI](https://api.travis-ci.org/team5499/frc-template.svg)
```

Comment:
Can you make this a link to the travis project page?

Reply:
yea gimme a sec